src/components/subscriptions/expiration/SubscriptionExpirationModals.jsx

```diff
@@ -90,17 +91,32 @@ const SubscriptionExpirationModals = ({ enterpriseId }) => {
     }
   }, [isSubscriptionExpired]);
 
+  const emitExpirationModalActionEvent = (dismiss) => {
```

Comment:
[nit/suggestion] Similar as above. It might be worth splitting this function into two: `emitModalActionEvent` and `emitModalDismissedEvent` so it's a bit more readable at a glance without having to decipher the `true/false` flag.